infrastructure/infra-aws.tf

```diff
@@ -96,6 +125,7 @@ resource "aws_elasticsearch_domain" "es" {
     volume_size = 512
   }
 
+  // TODO: add this back for prod ${join(", ", formatlist("\"arn:aws:sts::%s:assumed-role/dss-elk-oidc-proxy/%s\"", var.account_id, var.es_email_principals))},
```

Comment:
whats going on here?

Reply:
The proxy isn't deployed so the elasticsearch access policy can't point to the assumed roles it creates. There may be a workaround for this in the near-term, but we need to figure out a real fix for this.